bug.md

```diff
@@ -0,0 +1,58 @@
+# Bug
+
+Is an error, flaw, failure or fault in a computer program or system that causes it to produce an incorrect or unexpected result, or to behave in unintended ways.
+
+## Class not found
```

Comment:
Is this rule truly necessary?
'cause will show a compilation error, right?

Reply:
depends the version of Caché/Ensemble, some versions don't verify the existe of classes.

Reply:
ok, cool